File: JavascriptQuestion/question29.js
var arr=[2,7,11,15]
var target=9

for(i=0;i<arr.length;i++){
    for(j=0;j<i;j++){
    if(arr[i]+arr[j]==target)
    console.log("Pair:",arr[i],arr[j])
    }
}

File: CLangQuestion/question29.c
#include <stdio.h>
#include <string.h>

int main() {
    
    int arr[]={2,7,11,15};
    int len=sizeof(arr)/sizeof(arr[0]);
    int i,j;
    int target=9;
    
    for(i=0;i<len;i++){
        for(j=i+1;j<len;j++){
            if(arr[i]+arr[j]==target){
                printf("Pair: %d %d",arr[i], arr[j]);
            }
        }
    }
    
    return 0;
}
